### DISCUSSION TITLE
Is there a way to cleanup the application settings and start fresh?

### DISCUSSION THREAD

**Discussion by rshettyami** [awslabs/aws-saas-boost#494] [Q&A]
Is there a way to clean up the application settings in SaaS boost and start fresh?

**Comment by PoeppingT** [comment]
Hi @rshettyami . There is no way to do this in the Admin UI, but you can manually go behind the scenes to change application configuration. You should NOT do this if you have running tenants, as existing services will be removed that those tenant environments will break. Delete any existing tenants before refreshing your AppConfig.

The Settings Service (responsible for managing AppConfig and other SaaS Boost environment settings) uses AWS SSM ParameterStore as it's backing datastore. Parameters used by SaaS Boost are prefixed with `/saas-boost/${environment}/`. Service configurations are stored there as JSON blobs under `/saas-boost/${environment}/app/${service}/SERVICE_JSON`. 

To completely wipe clean an existing AppConfig, just delete all parameters prefixed with `/saas-boost/${environment}/app/`. Upon reload the Settings Service will see that no parameters already exist and will assume that the AppConfig is empty. The easiest way to do this is to:
1. go to your AWS Console
2. in the search bar at the top, search for "Systems Manager", and click on it
3. in the pane on the left, under "Application Management", you should see "Parameter Store". click on that.
4. now, in the Parameter search bar, type in `Name : begins-with : /saas-boost/${environment}/app`, replacing `${environment}` with your environment name
5. select all filtered Parameters and hit delete

Hopefully this helps.

**Reply by rshettyami** [reply]
Thank you so much @PoeppingT
